packages/client/helpers/functional-test/run-tests.ts

```diff
@@ -50,13 +59,17 @@ async function main(): Promise<number | void> {
       NODE_EXTRA_CA_CERTS: miniProxy.defaultCertificatesConfig.caCert,
     })
 
-    if (!args['--no-mini-proxy']) {
-      const qePath = await getBinaryForMiniProxy()
+    if (args['--edge-client']) {
+      jestCli = jestCli.withEnv({
+        DATA_PROXY_EDGE_CLIENT: 'true',
```

Comment:
Maybe rename to `TEST_DATA_PROXY_EDGE_CLIENT`, because it looks like a user facing env var at the moment

Reply:
done